### DISCUSSION TITLE
how to convert float to int in qweb report?

### DISCUSSION THREAD

**Discussion by it12uw** [OCA/odoo-community.org#132] [Q&A] [answered]
hello there,
i need help for converting the float to int in my odoo qweb report.

`<span t-value="int(doc.presentasi_total)"/> %%`

im using this , but there's nothing show on report, it only show the % symbol instead of the value.

any idea how to convert the float to int ?

thankyou

**Comment by luisg123v** [comment, marked as answer]
You should use `t-out` (or `t-esc` for older versions). `t-value` is not used to display values but for setting variables in combination with `t-set`.

**Reply by it12uw** [reply]
i'm using t-esc and its only show 0 %
but when i try this code ` <span t-esc="int(doc.total_mesin_jalan * 100)"/> %%` the value showing up.

**Reply by yostashiro** [reply]
How about `<span t-esc="'{:,.0f}'.format(doc.total_mesin_jalan * 100)" />`?

**Reply by it12uw** [reply]
its working too
thankyou
